deploy/kubelet-config/config.yaml

```diff
@@ -6,6 +6,3 @@ selectorSyncSet:
     - key: hive.openshift.io/version-major-minor
       operator: NotIn
       values: ["4.1", "4.2", "4.3", "4.4", "4.5", "4.6", "4.7", "4.8"]
-    - key: ext-managed.openshift.io/disable-debugging-handlers
```

Comment:
What's the significance of removing this label selector? I feel like there's low impact because of the machineConfigPoolSelector, but just want to make sure.

Reply:
This will always apply to clusters that match, 4.9+, and applies to all nodes.
The config for debugging handlers is in new KubeletConfig unique to the MCP.
MCO renders machineconfig taking all these into account.